language_service/src/completion/global_items.rs

```diff
@@ -281,6 +281,10 @@ impl<'a> Globals<'a> {
                     if !is_user_package && candidate_ns == ["Main".into()] {
                         return None;
                     }
+                    // filter out QASM namespaces
+                    if !is_user_package && namespace.name().to_lowercase().contains("qasm") {
```

Comment:
I think I'd feel safer with something a little more specific. If the name started with "Std.Qasm." I think would be a safer check. I worry that the check as-is could unintentionally trigger on namespaces in user-defined library packages.

Reply:
I left it as-is waiting for naming bikeshedding to be finished